### PR TITLE
Update the incompressible cylinder test case.

### PR DESCRIPTION
**Pull request by nloppi**

### CODE REVIEW

File: examples/inc_cylinder_2d/inc_cylinder_2d.ini [hunk 1]
```diff
@@ -49,35 +52,34 @@ soln-pts = williams-shunn
 soln-pts = gauss-legendre
 
 [soln-plugin-nancheck]
-nsteps = 50
+nsteps = 1
```
Comment (FreddieWitherden) [inline examples/inc_cylinder_2d/inc_cylinder_2d.ini]: Seems a bit frequent?

File: examples/inc_cylinder_2d/inc_cylinder_2d.ini [hunk 2]
```diff
@@ -49,35 +52,34 @@ soln-pts = williams-shunn
...
 
 [soln-plugin-pseudostats]
 flushsteps = 20
 file = residual.csv
 header = true
 
 [soln-plugin-writer]
-dt-out = 5.0
+dt-out = 1.0
```
Comment (FreddieWitherden) [inline examples/inc_cylinder_2d/inc_cylinder_2d.ini]: Seems a bit frequent?

**Comment by nloppi** [comment]
Reverted back to the original nstep values.

File: examples/inc_cylinder_2d/inc_cylinder_2d.ini [hunk 3]
```diff
@@ -6,11 +6,11 @@ rank-allocator = linear
 ;cblas = Enter path to local BLAS library for OpenMP backend
 
 [constants]
-nu = 0.008
+nu = 0.005
```
Comment (FreddieWitherden) [inline examples/inc_cylinder_2d/inc_cylinder_2d.ini]: Do we need to change the documentation to reflect this?

Comment (nloppi) [reply examples/inc_cylinder_2d/inc_cylinder_2d.ini]: I don't think we mention the Reynolds number anywhere. However, the figure needs to be changed. 

I will update the documention. Do you want a separate commit or shall I push everything here and rename the commit.?

Comment (FreddieWitherden) [reply examples/inc_cylinder_2d/inc_cylinder_2d.ini]: A separate commit is fine.

File: pyfr/integrators/dual/pseudo/pseudocontrollers.py [hunk 4]
```diff
@@ -121,7 +121,8 @@ def __init__(self, *args, **kwargs):
         tplargs['maxf'] = self.cfg.getfloat(sect, 'max-fact', 1.01)
         tplargs['minf'] = self.cfg.getfloat(sect, 'min-fact', 0.98)
         tplargs['saff'] = self.cfg.getfloat(sect, 'safety-fact', 0.8)
-        tplargs['dtau_maxf'] = self.cfg.getfloat(sect, 'dtau-max-fact', 3.0)
+        tplargs['dtau_maxf'] = self.cfg.getfloat(sect, 'pseudo-dt-max-fact',
```
Comment (nloppi) [inline pyfr/integrators/dual/pseudo/pseudocontrollers.py]: This had to be renamed for consistency.

**Pull request merged**